index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-header">
+        <div class="icons-section">
+            <div class="time">
+                <img alt="time icon" src="images/time.png">
+            </div>
+            <div class="group-icons">
+                <img alt="icon of cellural network,wifi and battery" src="images/group.png">
+            </div>
+        </div>
+        <nav class="nav-bar">
+            <a href="#" class="logo-name">Seth Bizimana</a>
+           <a href="#" class="menu-icon">
+                <img alt="icon of menu button" src="images/Union.png">
+           </a> 
+        </nav>
+    </header>
+    <main class="main">
+        
+        
+        <article class="headline">
+            <header class="main-header">
+                <span>Hello, This is Seth</span> 
+                <h2>I'am a Software Developer</h2>
+            </header>
+            <p class="headline-desc">
+                Ican help you build a product, feature or website
+                through somem of my work and experience! 
+                if you like what you see and intereseted to work
+                with me, reach me via:
+            </p>
+            <div class="social-icons">
+                <img alt="social media icons" src="images/social_media.png">
+            </div>
```

Comment:
- [ ] As said by the previous reviewer the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). So kindly update it.